content/questions/actions/question-093.md

```diff
@@ -0,0 +1,13 @@
+---
+archetype: "questions"
+title: "Question 093"
+question: "Which of the following is true about self-hosted runners in GitHub Actions?"
+draft: false
+---
+
+
+> https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners/about-self-hosted-runners 
+1. [ ] They are free for all repositories
```

Comment:
Aren't self hosted runners free for all repositories?

I know that you have to pay for your machines, but let's not make such tricky questions if that was the intention

Reply:
Got your point, let me exclude that tricky question